app/app.go

```diff
@@ -1012,6 +1016,9 @@ func (app *App) ProcessProposalHandler(ctx sdk.Context, req *abci.RequestProcess
 }
 
 func (app *App) FinalizeBlocker(ctx sdk.Context, req *abci.RequestFinalizeBlock) (*abci.ResponseFinalizeBlock, error) {
+	//TODO: update with logic that asserts that occ is enabled
+	ctx = ctx.WithIsOCCEnabled(true)
```

Comment:
can we move this to the `process block` function so its not in FinalizeBlocker and ProcessProposalHandler?